week-2/G-conditionals-2/exercise-3.js

```diff
@@ -9,7 +9,10 @@
 */
 
 function calculateGrade(mark) {
-
+  if (mark >= 80) return "A";
+  else if (mark >= 60 && mark < 80) return "B";
+  else if (mark >= 50 && mark < 60) return "C";
+  else if (mark < 50) return "Fail";
```

Comment:
This is good, a significant improvement from the previous exercises on conditionals. :+1: 

I would still improve this by removing the last if, as it is uneccessary (if it reached the last else condition then you already know that it is less than 50 so no need to have another check). Instead of a "else if", it can simply be an "else".